src/app/components/Footer/README.md

```diff
@@ -0,0 +1,24 @@
+# Footer
```

Comment:
```suggestion
# Footer [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind= Footer)
```

It'd be nice if this matched the precedence set by the [Header README](https://github.com/BBC-News/simorgh/blob/c8dad1f1c1bd5acc29f79588196c314cd164f8a1/src/app/components/Header/README.md).